docs/cvl/statements.md

```diff
@@ -156,6 +164,8 @@ Function calls
 This feature is currently undocumented.  See {ref}`call-expr` for partial information.
 ```
 
+- [`withrevert` example](https://github.com/Certora/Examples/blob/14668d39a6ddc67af349bc5b82f73db73349ef18/CVLByExample/storage/certora/specs/storage.spec#L45C19-L45C19)
+
```

Comment:
remove this todo and section and move the example to :
https://certora-certora-prover-documentation--156.com.readthedocs.build/en/156/docs/cvl/expr.html#calling-contract-functions

Reply:
This link is not in the documentation repo I moved the link to expr.md.